roofit/roofitcore/inc/TestStatistics/likelihood_builders.h

```diff
@@ -0,0 +1,60 @@
+// Author: Patrick Bos, Netherlands eScience Center / NIKHEF 2021
+
+/*****************************************************************************
+ * RooFit
+ * Authors:                                                                  *
+ *   WV, Wouter Verkerke, UC Santa Barbara, verkerke@slac.stanford.edu       *
+ *   DK, David Kirkby,    UC Irvine,         dkirkby@uci.edu                 *
+ *                                                                           *
+ * Copyright (c) 2000-2021, Regents of the University of California          *
+ *                          and Stanford University. All rights reserved.    *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef ROOT_ROOFIT_TESTSTATISTICS_likelihood_builders
+#define ROOT_ROOFIT_TESTSTATISTICS_likelihood_builders
+
+#include <TestStatistics/RooAbsL.h>
+#include <TestStatistics/optional_parameter_types.h>
+
+// forward declarations
+class RooAbsPdf;
+class RooAbsData;
+
+namespace RooFit {
+namespace TestStatistics {
+
+std::shared_ptr<RooAbsL>
```

Comment:
Here, it's very important that free functions that sit in this namespace are well documented (unless that comes in the cxx, then sorry for the comment).